### PR TITLE
Only build new docs if Quarkus is enabled in build

### PR DESCRIPTION
**Pull request by stianst**

### CODE REVIEW

**Review by miquelsi — DISMISSED** [review]

File: docs/pom.xml [hunk 1]
```diff
@@ -29,8 +29,17 @@
     <artifactId>keycloak-docs-parent</artifactId>
     <packaging>pom</packaging>
 
-    <modules>
-        <module>maven-plugin</module>
-        <module>guides</module>
-    </modules>
+    <profile>
```
Comment (mabartos) [inline docs/pom.xml:32]: Missing tag `<profiles>`

**Review by mabartos — CHANGES_REQUESTED** [review]

File: docs/pom.xml (hunk 1 above)
Comment (stianst) [reply docs/pom.xml]: Are you saying that I have to provide a valid pom file? ;)

**Comment by stianst** [comment]
Will squash this one on merge

**Review by DGuhr — APPROVED** [review]

File: docs/pom.xml (hunk 1 above)
Comment (mabartos) [reply docs/pom.xml]: Exactly :grin:

**Review by miquelsi — APPROVED** [review]

File: docs/pom.xml (hunk 1 above)
Comment (stianst) [reply docs/pom.xml]: Should be fixed now, can you check and approve if it's okay?

Comment (mabartos) [reply docs/pom.xml]: Great progress! Well done!

**Review by mabartos — APPROVED** [review]

**Review by pedroigor — APPROVED** [review]

**Pull request merged**